src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -583,6 +586,15 @@ VariantContext finalizeCurrentVariant(final List<VariantContext> unmergedVariant
         vcWithRef.genotypes(genotypes);
         mergedVC = vcWithRef.make();
 
+        if (maxAlternateAlleleCount.isPresent()) {
+            if (mergedVC.getAlternateAlleles().size() > maxAlternateAlleleCount.getAsLong()) {
+                    logger.warn("Dropping site on contig {} pos [{}, {}] with {} alt alleles versus maximum {}",
+                            mergedVC.getContig(), mergedVC.getStart(), mergedVC.getEnd(),
+                            mergedVC.getAlternateAlleles().size(), maxAlternateAlleleCount.getAsLong());
```

Comment:
the end position feels unnecessary here? but who am I to complain about more information?

Reply:
Oh right because we normalize everything at the site to the same ref? Welp, free end position info!